.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
     -   id: check-added-large-files
         args: ['--maxkb=1000']
     -   id: end-of-file-fixer
-        exclude: '^(.*\.svg)$'
+        exclude: '^(.*\.svg|docs/api-overview\.md)$'
```

Comment:
Can you add a comment to explain why api-overview is excluded?

Reply:
So since this .md file is generated, we don't expect it get updated manually and conflict with generated version. The linter pre-commit hook fails with `end-of-file-fixer` error. Updating the file manually results in failure of `Generate api docs` hook.

Reply:
But end-of-file-fixer failing is still valid right? Even if the content of this .md is auto-generated.

Reply:
You are right, cleaned up the generated file, and it fixed the issues.